doc/guides/abstraction/posix.rst

```diff
@@ -0,0 +1,434 @@
+
+
+POSIX Support
+#############
+
+The Portable Operating System Interface (POSIX) is a family of standards
+specified by the IEEE Computer Society for maintaining compatibility between
+operating systems. Zephyr implements a subset of the embedded profiles PSE51
+and PSE52 and BSD Sockets.
```

Comment:
> Zephyr implements a subset of the embedded profiles PSE51
and PSE52 and BSD Sockets.

I guess it would be better to say "Zephyr implements a subset of the embedded profiles PSE51, PSE52, and BSD Sockets API." cc: @dbkinder 

It would be also cool to provide links to all these stuffs, but I guess that can be done later.